docs/serving/knative-kubernetes-services.md

```diff
@@ -12,43 +6,41 @@ that are active when running Knative Serving.
 
 ## Before You Begin
 
-1. This guide assumes that you have installed Knative Serving. If you have not,
-   instructions on how to do this are located
-   [here](https://knative.dev/docs/install/).
+1. This guide assumes that you have installed Knative Serving. If you have not, instructions on how to do this are located [here](https://knative.dev/docs/install/).
```

Comment:
```suggestion
1. This guide assumes that you have [installed Knative Serving](https://knative.dev/docs/install/).
```

Reply:
wait actually, why wouldn't we link to the .md path?

Reply:
Going to use the .md path here...